src/sdk/bucketspace_fs.js

```diff
@@ -26,6 +26,29 @@ const ACCOUNT_PATH = 'accounts';
 const ACCESS_KEYS_PATH = 'access_keys';
 const bucket_semaphore = new KeysSemaphore(1);
 
+const BUCKET_EVENT_ERROR_CODE = {
```

Comment:
loved this struct, a few comments -
1. can we move it to a new manage_nsfs_events_utils.js file?
2. can we have a similar struct for accounts? and for IO events?
3. In this new events utils let's try create a function that receives the code and the arguments and runs the dbg.events since we now have many large events blocks in our code that I think we can reduce to a single line function call.

Reply:
@romayalon regarding point 3, When we try to move the `dbg.event()` to util file, we need to pass most of the arguments as parameters and this will result in a very long method call with a small change in number of lines.
we need to pass code, event_type, message, description, severity, state arguments, because these aruments are dependent on error types.

Reply:
what I meant is to pre-define these in a struct and just pass the code and the arguments, WDYT?

Reply:
@romayalon ya, that is something I can try out. I need to replace duplicate event code with unique code.